app/assets/javascripts/admin/edition_publishing.js

```diff
@@ -217,3 +217,21 @@ jQuery(function($) {
 
   SetTopicsFromPolicy.init();
 }(jQuery));
+
+(function($) {
+  var $policyInput = $('#edition_policy_content_ids'),
+      $importantCheckbox = $('#edition_important');
+
+  var toggleCheckboxIfPoliciesSelected = function() {
+    if ($importantCheckbox.length > 0) {
```

Comment:
This checks that there is a checkbox and then always sets it to disabled. So if you have JS enabled you will never be able to check this checkbox.